lua/lean/infoview.lua

```diff
@@ -22,25 +27,31 @@ local _DEFAULT_WIN_OPTIONS = {
 }
 local _NOTHING_TO_SHOW = { "No info found." }
 
---- An individual infoview.
+--- An individual info.
+local Info = {}
+local next_info_id = 0
+
+--- A "view" on an info (i.e. window).
 local Infoview = {}
+local next_infoview_id = 0
 
---- Get the ID of the infoview corresponding to the current window.
-local function get_id()
-  return vim.api.nvim_win_get_tabpage(0)
+--- Get the ID of the current infoview.
+function infoview.get_id()
```

Comment:
I'd very much rather this wasn't public. It seems like it's possibly just public for some testing reason, which I haven't fully understood yet.